src/pydvl/utils/utility.py

```diff
@@ -33,8 +37,9 @@ class Utility:
     """Convenience wrapper with configurable memoization of the scoring function.
 
     An instance of `Utility` holds the triple of model, dataset and scoring
-    function which determines the value of data points. This is mostly used for
-    the computation of :ref:`Shapley values<data valuation>`.
+    function which determines the value of data points. This is only used for
+    the computation of :ref:`Shapley values<Shapley>` and
+    :ref:`Least Core values<Least Core>`.
```

Comment:
"mostly" was future-proofing the docs: since we will soon add semivalues, we would have to remember to change the docstring is we use "only"

Reply:
Okay, I will change it to mostly.